application.py

```diff
@@ -80,4 +81,21 @@ def userhome():
         return render_template("home.html")
     else:
         flash("Login with the credentails.")
-        return render_template("register.html")
\ No newline at end of file
+        return render_template("register.html")
+
+@app.route('/book_page/<string:ISBN_number>', methods=["GET"])
+def book_page(ISBN_number):
+    if (ISBN_number == None):
+        flash("Invalid ISBN Number")
+        return render_template("book_page.html", books=None)
+    else:
+        if ISBN_number is None:
+            return None
```

Comment:
Is there any difference between "is None" and "== None"?

Reply:
changes made